components.go

```diff
@@ -133,7 +133,7 @@ func (cb *ComponentBase) SetAllDayEndAt(t time.Time, props ...PropertyParameter)
 func (cb *ComponentBase) getTimeProp(componentProperty ComponentProperty, expectAllDay bool) (time.Time, error) {
 	timeProp := cb.GetProperty(componentProperty)
 	if timeProp == nil {
-		return time.Time{}, errors.New("property not found")
+		return time.Time{}, ErrorPropertyNotFound
```

Comment:
It's good, however we can probably make it better with:
```
return time.Time{}, fmt.Errof("%w: %s", ErrorPropertyNotFound. componentProperty)
```

As it enhances it. I am very much in support of these updates.